src/main/resources/migration/changelog-v5.3.0.xml

```diff
@@ -0,0 +1,2298 @@
+            <column name="TEAM_ID"/>
+        </createIndex>
+
+        <createIndex indexName="NOTIFICATIONRULE_PROJECTS_N50" tableName="NOTIFICATIONRULE_PROJECTS">
+            <column name="PROJECT_ID"/>
+        </createIndex>
+
+        <createIndex indexName="NOTIFICATIONRULE_PROJECTS_N49" tableName="NOTIFICATIONRULE_PROJECTS">
+            <column name="NOTIFICATIONRULE_ID"/>
+        </createIndex>
+
+        <createIndex indexName="APIKEYS_TEAMS_N49" tableName="APIKEYS_TEAMS">
+            <column name="TEAM_ID"/>
+        </createIndex>
+
+        <createIndex indexName="APIKEYS_TEAMS_N50" tableName="APIKEYS_TEAMS">
+            <column name="APIKEY_ID"/>
+        </createIndex>
+
+        <createIndex indexName="PROJECT_ACCESS_TEAMS_N50" tableName="PROJECT_ACCESS_TEAMS">
+            <column name="TEAM_ID"/>
+        </createIndex>
+
+        <createIndex indexName="PROJECT_ACCESS_TEAMS_N49" tableName="PROJECT_ACCESS_TEAMS">
+            <column name="PROJECT_ID"/>
+        </createIndex>
+
+        <createIndex indexName="MANAGEDUSERS_TEAMS_N50" tableName="MANAGEDUSERS_TEAMS">
+            <column name="TEAM_ID"/>
+        </createIndex>
+
+        <createIndex indexName="MANAGEDUSERS_TEAMS_N49" tableName="MANAGEDUSERS_TEAMS">
+            <column name="MANAGEDUSER_ID"/>
+        </createIndex>
+
+        <createIndex indexName="POLICY_PROJECTS_N49" tableName="POLICY_PROJECTS">
+            <column name="POLICY_ID"/>
+        </createIndex>
+
+        <createIndex indexName="POLICY_PROJECTS_N50" tableName="POLICY_PROJECTS">
+            <column name="PROJECT_ID"/>
+        </createIndex>
+
+        <createIndex indexName="POLICY_TAGS_N50" tableName="POLICY_TAGS">
+            <column name="TAG_ID"/>
+        </createIndex>
+
+        <createIndex indexName="POLICY_TAGS_N49" tableName="POLICY_TAGS">
+            <column name="POLICY_ID"/>
+        </createIndex>
+
+        <addUniqueConstraint columnNames="GROUPNAME, PROPERTYNAME" constraintName="CONFIGPROPERTY_U1"
+                             tableName="CONFIGPROPERTY"/>
+
+        <createTable tableName="shedlock">
+            <column name="name" type="VARCHAR(64)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="shedlock_pk"/>
+            </column>
+            <column name="lock_until" type="TIMESTAMP WITHOUT TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="locked_at" type="TIMESTAMP WITHOUT TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="locked_by" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint baseColumnNames="VULNERABILITY" baseTableName="AFFECTEDVERSIONATTRIBUTION"
+                                 constraintName="AFFECTEDVERSIONATTRIBUTION_FK1" deferrable="true"
```

Comment:
Only looking at this just now. Should we also make the foreign key names more predictable? I am not sure how consistent DataNucleus' naming is / was.

Reply:
We can rename it to CurrentTable_ReferencedTable_FK. 
Found this query somewhere to give us all constraint information we need 

`SELECT
    tc.table_schema, 
    tc.constraint_name, 
    tc.table_name, 
    kcu.column_name, 
    ccu.table_schema AS foreign_table_schema,
    ccu.table_name AS foreign_table_name,
    ccu.column_name AS foreign_column_name 
FROM information_schema.table_constraints AS tc 
JOIN information_schema.key_column_usage AS kcu
    ON tc.constraint_name = kcu.constraint_name
    AND tc.table_schema = kcu.table_schema
JOIN information_schema.constraint_column_usage AS ccu
    ON ccu.constraint_name = tc.constraint_name
WHERE tc.constraint_type = 'FOREIGN KEY';`